CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 1.1.0
+
+* Add support for animated tiles in tilesets (Ashley Davis (SgtCoDFish))
```

Comment:
I'll make sure to add this line when I tag a new release, but could you leave it out of the PR?